calcom/cal.com
# Requires Confirmation Threshold

## What does this PR do?

Introduces the ability to add a threshold to the requires confirmation event type option.

Closes #3964

Loom Video: https://www.loom.com/share/e197de9655e0401c82673d6609107ed9

**Environment**: Staging(main branch) / Production

## Type of change


## How should this be tested?

Go to an event type configuration page under Advanced tab and check Requires Confirmation. It will show up 2 options, "Always" or to set the threshold.
